docs/source/user-guide/install/linux.rst

```diff
@@ -44,7 +44,13 @@ Using with fish shell
 
 To use conda with fish shell, run the following in your terminal:
 
-  ``conda init fish``
+ #. Add conda binary to $PATH, if not yet added::
+
+      fish_add_path path/to/condabin
```

Comment:
```suggestion
      fish_add_path <conda-install-location>/condabin
```

I think this makes it a little more clear that the `condabin` folder is located where conda itself was installed.